### PR TITLE
Visual improvements + clean up commands

### PR DESCRIPTION
**Pull request by sdan**
BRING THE COLORS

### CODE REVIEW

File: ui/text.go [hunk 1]
```diff
@@ -15,3 +15,23 @@ func Bold(text string) string {
 	color := Color(os.Stdout)
 	return color.Sprintf(color.Bold(text))
 }
+
+func RedText(payload string) aurora.Value {
+	color := Color(os.Stdout)
+	return color.Red(payload).Bold()
```
Comment (coffee-cup) [inline ui/text.go:21]: Should all the colours be bold as well? IMO the `RedText` function should only change the color.

File: ui/text.go [hunk 2]
```diff
@@ -15,3 +15,23 @@ func Bold(text string) string {
 	color := Color(os.Stdout)
 	return color.Sprintf(color.Bold(text))
 }
+
+func RedText(payload string) aurora.Value {
+	color := Color(os.Stdout)
```
Comment (coffee-cup) [inline ui/text.go:20]: Why is `os.Stdout` needed? Based on the aurora docs, would this work instead?

```go
func RedText(payload string) aurora.Value {
	return aurora.Red(payload)
}
```

**Review by coffee-cup — COMMENTED** [review]
This is definitly an improvement. I do think that there is not enough consistency with the styles. For example,

image

In the above,
- the list select finish styles are different
- not sure why the text "Connected" is bold and purple

IMO (feel free to disagree), list select styles should finish like in "Connected to existing project" as we don't want to draw attention to them. Colors should not also be bold as well. And only a single section of text should be coloured per line as it should be used to draw the users attention to something specific.

**Comment by JakeCooper** [comment]
Definitely fuck with the pink, but we should probs unify this

Before:
Screen Shot 2020-10-19 at 6 17 24 PM

After:
<green>Project:</green><pink>project.name</pink>
<green>Environment:

**Comment by JakeCooper** [comment]
🤦

File: cmd/up.go [hunk 3]
```diff
@@ -2,10 +2,21 @@ package cmd
...
+	"fmt"
 
 	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
 )
 
 func (h *Handler) Up(ctx context.Context, req *entity.CommandRequest) error {
-	return h.ctrl.Up(ctx)
+	ui.StartSpinner(&ui.SpinnerCfg{
+		Message: "Accelerating your train to the clouds...",
```
Comment (JakeCooper) [inline cmd/up.go:13]: Laying tracks in the clouds?

**Review by JakeCooper — APPROVED** [review]
Thumb

**Pull request merged**